src/main/java/thePackmaster/cards/contentcreatorpack/Wanderbots.java

```diff
@@ -0,0 +1,42 @@
+package thePackmaster.cards.contentcreatorpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.defect.ChannelAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.orbs.AbstractOrb;
+import com.megacrit.cardcrawl.orbs.EmptyOrbSlot;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.orbs.contentcreatorpack.Wanderbot;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+
+public class Wanderbots extends AbstractContentCard {
+    public final static String ID = makeID("Wanderbots");
+
+    public Wanderbots() {
+        super(ID, 1, CardType.SKILL, CardRarity.UNCOMMON, CardTarget.SELF);
+        baseBlock = 6;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        blck();
+        atb(new ChannelAction(new Wanderbot()));
+        if (upgraded) atb(new AbstractGameAction() {
+            @Override
+            public void update() {
+                isDone = true;
+                for (AbstractOrb o : AbstractDungeon.player.orbs) {
+                    if (!(o instanceof EmptyOrbSlot)) {
+                        o.onEndOfTurn();
```

Comment:
Would it make sense to use `InscribedMod` instead of this code? This might have been written before that was in the repo, not sure.

Even if you don't use `InscribedMod`, you should copy over how it activates all different orbs types. Calling `onStartOfTurn` is needed for orbs like Plasma, and checking for `PackmasterOrb` and calling `passiveEffect` is necessary to support orbs that only have a passive effect (that is only triggered by cards like this, not start or end of turn).
```
        for (AbstractOrb o : AbstractDungeon.player.orbs){
            if (o instanceof PackmasterOrb){
                ((PackmasterOrb) o).passiveEffect();
            } else {
                o.onStartOfTurn();
                o.onEndOfTurn();
            }
        }
```
(The code above is pasted from `InscribedMod` and might benefit from a check for `EmptyOrbSlot`.)

Reply:
I'll swap it to use InscribedMod for consistency, thanks!

Reply:
Also added a check for emptyorbslot in the InscribedMod and added Inherent functionality.